### ISSUE TITLE
docs list March twice, miss Feb

### ISSUE THREAD

**Issue by dimitropoulos** [clauderic/react-infinite-calendar#140]
![image](https://user-images.githubusercontent.com/15232461/30327631-c2b3d958-979a-11e7-996c-c3046dbf36bb.png)

http://clauderic.github.io/react-infinite-calendar/#/customization/select-year-first?_k=0pth0m

**Comment by clauderic** [comment]
What browser are you using / which timezone are you in?

**Comment by michaelolo24** [comment]
I'm getting the same error in the US (east coast)

**Comment by dimitropoulos** [comment]
chrome in ET

**Comment by talibasya** [comment]
I had the same when `selected` was `moment.toString()` value.
Helped replacing by native js `Date` object 😎

**Comment by meta-meta** [comment]
I believe this is the issue https://github.com/clauderic/react-infinite-calendar/blob/master/src/Years/index.js#L53

What does the selected date have to do with months in the year?

**Comment by leron8** [comment]
Hey we're also having this issue! I see that @mvayngrib committed a fix. Does that solve it.? It hasn't been merged for a release right?

**Comment by meta-meta** [comment]
@leron8 I've had a PR open for 8 months. https://github.com/clauderic/react-infinite-calendar/issues/150
It fixes the issue.

**Comment by leron8** [comment]
Thanks @meta-meta , how could I use that fix? would I have to fork this repo and build it myself or is the only option to wait for the pull request to be merged?

**Comment by meta-meta** [comment]
@leron8 Yes, I believe that's the only thing you can do until it's merged and published. If you want, feel free to use my fork by adding this to your package.json: `    "react-infinite-calendar": "meta-meta/react-infinite-calendar#built",`

**Comment by leron8** [comment]
great @meta-meta I see that it's the only change you comparing to the original repo right? https://github.com/clauderic/react-infinite-calendar/compare/master...meta-meta:master ... well besides this: https://github.com/clauderic/react-infinite-calendar/commit/d32f20c6e26bfcca2fb67854c90f4d82a91c3cb6 but it only changes README.md
